Unable to build with wxWidgets 3.0.2 (wxGTK3)

Log:
```
cd /home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/wx-config-3.0/x86_64-redhat-linux-gnu/src && /usr/bin/c++  -DWXUSINGDLL -D_FILE_OFFSET_BITS=64 -D__WXGTK__ -I/home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/wx-config-3.0/x86_64-redhat-linux-gnu/src -isystem /usr/lib64/wx/include/gtk3-unicode-3.0 -isystem /usr/include/wx-3.0  -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -pthread -g   -o CMakeFiles/wxmaxima.dir/main.cpp.o -c /home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/main.cpp
/home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/Worksheet.cpp: In member function 'void Worksheet::OnPaint(wxPaintEvent&)':
/home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/Worksheet.cpp:433:30: error: no matching function for call to 'wxAutoBufferedPaintDC::SetPen(const wxPen*)'
   dc.SetPen(wxTRANSPARENT_PEN);
                              ^
/home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/Worksheet.cpp:433:30: note: candidate is:
In file included from /usr/include/wx-3.0/wx/wx.h:50:0,
                 from /home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/wxMaximaFrame.h:33,
                 from /home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/wxMaxima.h:34,
                 from /home/scx/rpmbuild/BUILD/wxmaxima-Version-19.05.0/src/Worksheet.cpp:32:
/usr/include/wx-3.0/wx/dc.h:881:10: note: void wxDC::SetPen(const wxPen&)
     void SetPen(const wxPen& pen)
          ^
/usr/include/wx-3.0/wx/dc.h:881:10: note:   no known conversion for argument 1 from 'const wxPen*' to 'const wxPen&'
```

Temporary patch:
```
diff --git a/src/Worksheet.cpp b/src/Worksheet.cpp
index 37fccf6..7c63e4b 100644
--- a/src/Worksheet.cpp
+++ b/src/Worksheet.cpp
@@ -430,7 +430,7 @@ void Worksheet::OnPaint(wxPaintEvent &WXUNUSED(event))
   dc.Clear();
 #else
 #ifdef __WXGTK3__
-  dc.SetPen(wxTRANSPARENT_PEN);
+  dc.SetPen(*wxTRANSPARENT_PEN);
   dc.DrawRectangle(updateRegion);
 #else
   dc.Clear();
```

PR is on the way.

--- a/src/Worksheet.cpp
+++ b/src/Worksheet.cpp
@@ -430,7 +430,7 @@ void Worksheet::OnPaint(wxPaintEvent &WXUNUSED(event))
   dc.Clear();
 #else
 #ifdef __WXGTK3__
-  dc.SetPen(wxTRANSPARENT_PEN);
+  dc.SetPen(*wxTRANSPARENT_PEN);
   dc.DrawRectangle(updateRegion);
 #else
   dc.Clear();

